app/common/state/tabContentState/audioState.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Utils
+const {getFrameByKey} = require('../../../../js/state/frameStateUtil')
+const {isEntryIntersected} = require('../../../../app/renderer/lib/observerUtil')
+
+module.exports.canPlayAudio = (state, frameKey) => {
+  const frame = getFrameByKey(state, frameKey)
+
+  if (frame == null) {
+    if (process.env.NODE_ENV !== 'test') {
```

Comment:
I see that you use this pattern a lot. I am curious why we wouldn't like to see this error in `test` as well?

Reply:
it always falls to that condition for unit tests even when the test pass for some async reason

Reply:
ok will check out unit tests and try to fix it, so that we can remove this check